src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -41,7 +42,10 @@ public CommandResult execute(Model model) throws CommandException {
 
         Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
         model.deletePerson(personToDelete);
-        return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, personToDelete));
+        if (personToDelete instanceof Patient) {
```

Comment:
Would it be better if we implemented isPatient() and isNurse() methods in the Person class?

Reply:
ok noted, I will update it

Reply:
> IMO getCategory() would be preferable to isPatient() and isNurse() because:
> 
> 1. we would need 2 methods to essentially give us the same information
> 2. getCategory would be more scalable in the future, in case we want to add more categories such as Doctor, Social Worker etc.
> 
> Thoughts?

Actually when I was making the class Category, I was thinking if we should include getCategory() inside Person. However, after much thinking, I feel like category shouldn't be inside person and I just stuck with instance of

Reply:
I agree that category should be its own class and not inside person. What's wrong with having a Person::getCategory() though?